domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/service/EntityToResponseDTOMapper.java

```diff
@@ -221,6 +221,8 @@ private EventComplexTypeEntityToCaseEventFieldComplexListMapper() {
     @Mapping(source = "searchAliasFieldEntity.caseType.reference", target = "caseTypeId")
```

Comment:
commenting here because it doesn't allow me to comment on the spreadsheet files. 
I see we have added many new test spreadsheets. In my opinion this should be avoided. The reason being that this increases that maintainability effort of our tests. Because spreadsheet change, we might add new mandatory elements in the future. And when that happens we'll have to update all the test spreadsheets. 
Do we need to add CCD_TestDefinition_V46_RDM-4324.xlsx? Can't we update an existing one?
Also, I see many spreadsheets probably added to test corner case conditions in func tests or integration tests. Do we need to use func tests or int tests for that? We should be able to test all the corner cases with unit tests. To reduce the amount of int and func test to the bare minimun.

Reply:
done changes